src/writer.js

```diff
@@ -246,7 +250,10 @@ define([
                 opts.writeCustomXML(xmlWriter, mug);
             }
             if (opts.writeControlRefAttr) {
-                xmlWriter.writeAttributeString(opts.writeControlRefAttr, mug.absolutePath);
+                var hashtag = mug.hashtagPath;
+                if (hashtag) {
```

Comment:
https://github.com/dimagi/Vellum/commit/31532cab72afd06d95cdad25d1550c3773cfd3c8

xpath fails when you pass in `""`, `null`, `undefined`, but we have various places where those are legitamate values that are passed in. `xmlWriter.writeAttributeString` handled these values by not writing